src/clarity-angular/layout/vertical-nav/_vertical-nav.clarity.scss

```diff
@@ -0,0 +1,292 @@
+//General
```

Comment:
i am trying to follow suitcss guidelines that @mathisscott and I discussed a couple of months back. for eg: notice the separation between `display`, `dimensions` & `other properties` in each class.

good to know that you liked it and i think we should follow this for our other components too. if you think we can improve it further let me know.